### PR TITLE
Project structure and optimizations

### PR DESCRIPTION
**Pull request by mwigton**
Organized project and added optimizations

Notable points:
- Removed binaries, should not be in repo
- Moved resources to own folder
- Resized background image, large performance increase (load time)
- Fixed upload file filter, not possible to select file (on windows) 
- Removed makefile, is generated by qmake per system

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,2 @@
 # Typometer
 Basic GUI app to test your typing speed made on QT framework.
-
-If you get any compilation errors please have a look to : http://wiki.qt.io/Transition_from_Qt_4.x_to_Qt5
```
Comment (himanshukumar660) [inline README.md:4]: Compilations issues had been obserevd with Qt4.

**Review by himanshukumar660 — COMMENTED** [review]
Nice work. But still certain things need to be done.

File: README.md (hunk 1 above)
Comment (mwigton) [reply README.md:4]: Ah, didn't know you were testing with Qt4 as well, makes sense. Might be a good idea to specify in the readme which qt version and compiler is known to work? I used Qt 5.7 with MSVC2013

Comment (himanshukumar660) [reply README.md:4]: Maybe you could make a ReadMe file. ;)

Comment (mwigton) [reply README.md:4]: Sure, Ill do that tonight.

**Pull request merged**